templates/note/message_view.html

```diff
@@ -32,6 +32,13 @@
 
     <div class="formatted-content">$:{MARKDOWN(query['content'])}</div>
 
+    $if myself['is_mod']:
+      <form action="/modcontrol/copynotetostaffnotes" method="post" class="form clear">
+        <button class="button positive" style="float: right;">Copy Note to Staff Notes</button>
```

Comment:
Also probably doesn’t need a `positive`. There’s already one bold, green, primary button on the page.